esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1829,6 +1829,35 @@ bool CFileSprayEx::onGetDFUExceptions(IEspContext &context, IEspGetDFUExceptions
     return true;
 }
 
+static StringBuffer &getDropZoneHost(const char *planeName, IPropertyTree *plane, StringBuffer &host)
+{
+    if (plane->queryProp("hosts"))
```

Comment:
Shouldn't this be

    plane->hasProp("hosts")

?

Reply:
yes, I'll correct.